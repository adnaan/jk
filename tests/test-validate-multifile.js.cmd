jk validate -m ./validate-files/module ./testfiles/validate/validate-multi.yaml
