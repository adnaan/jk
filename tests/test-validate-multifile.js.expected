./testfiles/validate/validate-multi.yaml: object name is not "Valid"
